examples/platforms/cc2538/alarm.c

```diff
@@ -32,9 +32,6 @@
  *
  */
 
-#include <stdbool.h>
```

Comment:
Why the changes to the cc2538 files? Windows targets shouldn't be building these files.